custom_components/waste_collection_schedule/waste_collection_schedule/source/renodjurs_dk.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Reno Djurs"
+DESCRIPTION = "Reno Djurs collections for Djursland"
+URL = "https://renodjurs.dk"
+TEST_CASES = {
+    "TestCase1": {
+        "id": "46000",
+    },
+    "TestCase2": {
+        "id": "45000",
+    },
+    "TestCase3": {
+        "id": 44000,
+    },
+}
+
+ICON_MAP = {
+    "REST-MAD": "mdi:trash-can",
+    "PAP-PAPIR": "mdi:newspaper",
+    "PLAST-GLAS-METAL": "mdi:bottle-wine",
+}
+
+
+class Source:
+    def __init__(self, id: str | int):
+        self._id = str(id)
+        self._api_url = f"https://minside.renodjurs.dk/Ordninger.aspx?id={self._id}"
+
+    def fetch(self):
+        session = requests.Session()
+        
+        entries = []
+
+        r = session.get(self._api_url)
+        r.raise_for_status()
+        soup = BeautifulSoup(r.text, "html.parser")
+        container = soup.find("table", {"class": "table"})
+        rows = container.find_all("tr")
+        
+        for row in rows:
+            cells = row.find_all("td")
+            if len(cells) > 4:
```

Comment:
I'd prefer `if len(cells) >= 4: continue` to reduce indentation depth

Reply:
Guess you mean `<=` and not `>=`